notebooks/7-rotation-speeds/interm_rotation_rate_solutions.ipynb

```diff
@@ -0,0 +1,3039 @@
+{
```

Comment:
<p>Still feasible? To discuss during our meeting, I suppose</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>